cts/scheduler/summary/clone-recover-no-shuffle-8.summary

```diff
@@ -0,0 +1,52 @@
+Current cluster status:
+  * Node List:
+    * Online: [ node1 node2 node3 ]
+
+  * Full List of Resources:
+    * Fencing	(stonith:fence_xvm):	 Started node2
+    * Clone Set: grp-clone [grp] (promotable):
+      * Promoted: [ node2 ]
+      * Unpromoted: [ node3 ]
+      * Stopped: [ node1 ]
+
+Transition Summary:
+  * Demote     rsc1:1     ( Promoted -> Unpromoted node2 )
+  * Demote     rsc2:1     ( Promoted -> Unpromoted node2 )
```

Comment:
I just noticed that "demote rsc2" doesn't seem to be ordered after "demote rsc1" when demoting a group. That might be another bug.

Reply:
FYI I find the dot files helpful to get that at a glance. I have a convenience function for e.g. "simview -x clone-recover-no-shuffle.xml":

    simview () 
    { 
        local TMPDOT;
        local TMPSVG;
        if [ $# -gt 0 ]; then
            TMPDOT=$(mktemp ${TMPDIR:-/tmp}/simview.XXXXXXXXXX.dot);
            crm_simulate -S $@ -D "$TMPDOT";
            if [ $? -eq 0 ]; then
                TMPSVG=$(mktemp ${TMPDIR:-/tmp}/simview.XXXXXXXXXX.svg);
                dot "$TMPDOT" -Tsvg > "$TMPSVG";
                [ $? -eq 0 ] && eog "$TMPSVG";
                rm -f "$TMPSVG";
            fi;
            rm -f "$TMPDOT";
        else
            echo 'Usage: simview <crm_simulate-xml-input-option>';
        fi
    }

Reply:
This is very helpful. And the dot graph for this file shows the correct order dependency between demoting rsc2 and demoting rsc1.